Makefile

```diff
@@ -248,10 +250,12 @@ $(DIST_DIR)/natcap.invest%.zip: | $(DIST_DIR)
 # on Windows as the .exe extension is assumed.
 binaries: $(INVEST_BINARIES_DIR)
 $(INVEST_BINARIES_DIR): | $(DIST_DIR) $(BUILD_DIR)
-	-$(RMDIR) $(BUILD_DIR)/pyi-build
-	-$(RMDIR) $(INVEST_BINARIES_DIR)
+	-rm -r $(BUILD_DIR)/pyi-build
+	-rm -r $(INVEST_BINARIES_DIR)
 	$(PYTHON) -m PyInstaller --workpath $(BUILD_DIR)/pyi-build --clean --distpath $(DIST_DIR) exe/invest.spec
-	$(BASHLIKE_SHELL_COMMAND) "$(PYTHON) -m pip freeze --all > $(INVEST_BINARIES_DIR)/package_versions.txt"
+	# wrapping conda command in powershell since some windows bash shells
+	# don't have access to conda command.
+	powershell.exe -Command "conda list --export > $(INVEST_BINARIES_DIR)/package_versions.txt"
```

Comment:
I think this powershell reference is what is causing the Mac builds to fail right now.

Reply:
Oh yea, oops. Looks like the homebrew Mac bugs have been fixed though!

Reply:
Thanks for spotting this. I fixed it here: https://github.com/natcap/invest/pull/314/commits/fa4e458c6b9d02c6aeeb38b277176be67a78b0a9 by setting a `CONDA` environment variable that calls `conda` for mac/nix and `conda.bat` for windows.